components/breadcrumb/index.css

```diff
@@ -1,333 +1,261 @@
-
-  --spectrum-breadcrumbs-action-button-spacing-block-multiline: var(--spectrum-breadcrumbs-top-to-truncated-menu-compact); /* multiline */
-  --spectrum-breadcrumbs-action-button-spacing-block-between-multiline: var(--spectrum-breadcrumbs-truncated-menu-to-bottom-text); /* multiline */
-
-  /* Focus Indicator */
-  --spectrum-breadcrumbs-focus-indicator-thickness: var(--spectrum-focus-indicator-thickness);
-  --spectrum-breadcrumbs-focus-indicator-gap: var(--spectrum-focus-indicator-gap);
-
-  /* placeholder for border radius for focus indicator */
-  --spectrum-breadcrumbs-item-link-border-radius: var(--spectrum-corner-radius-100);
-
-  /* Colors */
-  --spectrum-breadcrumbs-text-color: var(--spectrum-neutral-subdued-content-color-default);
-  --spectrum-breadcrumbs-text-color-current: var(--spectrum-neutral-content-color-default);
-  --spectrum-breadcrumbs-text-color-disabled: var(--spectrum-disabled-content-color);
-  --spectrum-breadcrumbs-separator-color: var(--spectrum-neutral-content-color-default);
-  --spectrum-breadcrumbs-action-button-color: var(--spectrum-neutral-subdued-content-color-default);
-  --spectrum-breadcrumbs-action-button-color-disabled: var(--spectrum-disabled-content-color);
-
-  --spectrum-breadcrumbs-focus-indicator-color: var(--spectrum-focus-indicator-color);
-}
-
-/* windows high contrast mode */
-@media (forced-colors: active) {
-  .spectrum-Breadcrumbs {
-    --highcontrast-breadcrumbs-text-color: LinkText;
-    --highcontrast-breadcrumbs-text-color-current: CanvasText;
-    --highcontrast-breadcrumbs-text-color-disabled: GrayText;
-    --highcontrast-breadcrumbs-separator-color: CanvasText;
-    --highcontrast-breadcrumbs-action-button-color: LinkText;
-    --highcontrast-breadcrumbs-action-button-color-disabled: GrayText;
-
-    --highcontrast-breadcrumbs-focus-indicator-color: CanvasText;
-  }
-}
-
-.spectrum-Breadcrumbs {
-  list-style-type: none;
-
-  margin: 0;
-  padding-inline-start: var(--mod-breadcrumbs-inline-start, var(--spectrum-breadcrumbs-inline-start));
-  padding-inline-end: var(--mod-breadcrumbs-inline-end, var(--spectrum-breadcrumbs-inline-end));
-
-  display: flex;
-  flex-flow: row nowrap;
-  align-items: center;
-  justify-content: flex-start;
-  flex: 1 0 0%;
-
-  block-size: var(--mod-breadcrumbs-block-size, var(--spectrum-breadcrumbs-block-size));
+	--spectrum-breadcrumbs-block-size: var(--spectrum-component-size-100);
+
+	/* Text - medium / default */
+	--spectrum-breadcrumbs-line-height: var(--spectrum-line-height-100);
+	--spectrum-breadcrumbs-font-size: var(--spectrum-font-size-100);
+	--spectrum-breadcrumbs-font-family: var(--spectrum-sans-font-family-stack);
+	--spectrum-breadcrumbs-font-weight: var(--spectrum-regular-font-weight);
+
+	--spectrum-breadcrumbs-font-size-current: var(--spectrum-font-size-100);
+	--spectrum-breadcrumbs-font-family-current: var(--spectrum-sans-font-family-stack);
```

Comment:
Font-stack question: why does `"Adobe Clean"` render differently than `adobe-clean`? Using "Adobe Clean" looks more like the design, but our font stack renders adobe-clean because that's the first font in the stack.

Maybe not something that needs to be fixed, but wanted to point it out.

![Screenshot 2024-11-18 at 11 39 21 AM](https://github.com/user-attachments/assets/8ce7f77c-893d-45ff-9bf5-9e3cdc5dea26)